runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4001,23 +4001,51 @@ class INTERPRETER_CLASS
 		return EXECUTE_BYTECODE;
 	}
 
-	/* ToDo: unimplemented - https://github.com/eclipse-openj9/openj9/issues/13614 */
 	/* jdk.internal.misc.Unsafe: public native <V> V uninitializedDefaultValue(Class<?> clz); */
 	VMINLINE VM_BytecodeAction
 	inlUnsafeUninitializedDefaultValue(REGISTER_ARGS_LIST)
 	{
+		j9object_t cls = *(j9object_t*)_sp;
+
 		updateVMStruct(REGISTER_ARGS);
-		Assert_VM_unreachable();
+		buildInternalNativeStackFrame(REGISTER_ARGS);
```

Comment:
the buildInternalNativeStackFrame/restoreInternalNativeStackFrame are not needed in either of these INLS, likewise wth `updateVMStruct`.

Generally speaking, you need it if there is a possibility of losing VMAccess. In this case it will not happen.

Reply:
addressed in 3b30214